src/controllers/category.js

```
@@ -48,17 +48,19 @@ const getCategories = async (req, res) => {
 }
 
 const getSubjects = async (req, res) => {
-  const { name, limit, skip } = req.params
+  const { id } = req.params
+  const { name, limit, skip } = req.query
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)